lib/chili_project/principal_allowance_evaluator/base.rb

```diff
@@ -54,4 +54,8 @@ def global_granting_candidates
   def project_granting_candidates project
     []
   end
+
+  def self.eager_load_for_project_authorization(project)
```

Comment:
Unused method argument - `project`. If it's necessary, use `_` or `_project` as an argument name to indicate that it won't be used. You can also write as `eager_load_for_project_authorization(*)` if you want the method to accept any arguments but don't care about them.